test/jit/test_export_modes.py

```diff
@@ -68,6 +68,24 @@ def foo(a):
         x = torch.ones(3)
         torch.onnx._export(foo, (x,), f)
 
+    @skipIfNoCaffe2
+    @skipIfNoLapack
+    def test_caffe2_aten_fallback(self):
```

Comment:
Can this go in test_pytorch_onnx_caffe2.py instead?

Reply:
This is a `export mode` test file, which includes caffe2 as one of the modes. Are you sure we should move it?

Reply:
This doesn't have AssertONNX either?
In any case, I can see why this test makes sense to keep in this file, so I withdraw my request. This is OK to keep here.

Reply:
kept here